docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# base ubuntu 18.04 image
+FROM ubuntu:bionic
+
+## system-wide dependencies
+RUN apt-get update && \
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+build-essential git ffmpeg bc \
+python3 python3-dev python3-pip python3-setuptools python3-wheel
+
+# install video-server app
+WORKDIR /opt/video-server-app
+RUN git clone https://github.com/superdesk/video-server-app.git .
```

Comment:
I wonder why not use `ADD` if this file is already in the repo

Reply:
I think in this case I should move `Dockerfile` outside of the `docker` folder.

Reply:
yep right